k8s/neuvector/apptest/tester/tester.sh

```diff
@@ -17,6 +17,6 @@
 set -xeo pipefail
 shopt -s nullglob
 
-for test in /tests/*; do
```

Comment:
Personally, I would leave the original code in, with the exit 0 above it, to retain the testrunner plumbing example in the code.